vendor/assets/update-ui-kit.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+wget http://gov-au-ui-kit.apps.staging.digital.gov.au/latest/ui-kit.js -O javascripts/ui-kit.js
+wget http://gov-au-ui-kit.apps.staging.digital.gov.au/latest/_ui-kit.scss -O stylesheets/_ui-kit.scss
+wget http://gov-au-ui-kit.apps.staging.digital.gov.au/latest/ui-kit-icons.css -O stylesheets/ui-kit-icons.css
```

Comment:
Question: does it make more sense for this to be here or `/bin`? It's an executable file but it's to do with the assets...